src/pudl/output/pudltabl.py

```diff
@@ -129,6 +130,76 @@ def __init__(
+        """Load output assets and register a class method for retrieving each one."""
+        # Map table name to PudlTabl method.
+        # PudlTabl will generate a method to read each table from the DB with the given method name
+        table_method_map = {
+            # denorm_ferc1
+            "denorm_balance_sheet_assets_ferc1": "denorm_balance_sheet_assets_ferc1",
+            # denorm_eia
+            "denorm_utilities_eia": "denorm_utilities_eia",
+            # eia861
+            "service_territory_eia861": "service_territory_eia861",
+            "sales_eia861": "sales_eia861",
+            "advanced_metering_infrastructure_eia861": "advanced_metering_infrastructure_eia861",
+            "demand_response_eia861": "demand_response_eia861",
+            "demand_response_water_heater_eia861": "demand_response_water_heater_eia861",
+            "demand_side_management_sales_eia861": "demand_side_management_sales_eia861",
+            "demand_side_management_ee_dr_eia861": "demand_side_management_ee_dr_eia861",
+            "demand_side_management_misc_eia861": "demand_side_management_misc_eia861",
+            "distributed_generation_tech_eia861": "distributed_generation_tech_eia861",
+            "distributed_generation_fuel_eia861": "distributed_generation_fuel_eia861",
+            "distributed_generation_misc_eia861": "distributed_generation_misc_eia861",
+            "distribution_systems_eia861": "distribution_systems_eia861",
+            "dynamic_pricing_eia861": "dynamic_pricing_eia861",
+            "energy_efficiency_eia861": "energy_efficiency_eia861",
+            "green_pricing_eia861": "green_pricing_eia861",
+            "mergers_eia861": "mergers_eia861",
+            "net_metering_customer_fuel_class_eia861": "net_metering_customer_fuel_class_eia861",
+            "net_metering_misc_eia861": "net_metering_misc_eia861",
+            "non_net_metering_customer_fuel_class_eia861": "non_net_metering_customer_fuel_class_eia861",
+            "non_net_metering_misc_eia861": "non_net_metering_misc_eia861",
+            "operational_data_revenue_eia861": "operational_data_revenue_eia861",
+            "operational_data_misc_eia861": "operational_data_misc_eia861",
+            "reliability_eia861": "reliability_eia861",
+            "utility_data_nerc_eia861": "utility_data_nerc_eia861",
+            "utility_data_rto_eia861": "utility_data_rto_eia861",
+            "utility_data_misc_eia861": "utility_data_misc_eia861",
+            "utility_assn_eia861": "utility_assn_eia861",
+            "balancing_authority_eia861": "balancing_authority_eia861",
+            "balancing_authority_assn_eia861": "balancing_authority_assn_eia861",
+            # ferc714
+            "respondent_id_ferc714": "respondent_id_ferc714",
+            "demand_hourly_pa_ferc714": "demand_hourly_pa_ferc714",
+        }
+
+        for table_name, method_name in table_method_map.items():
+            if hasattr(PudlTabl, method_name):
```

Comment:
We can probably remove this at some point, but for now it will generate a warning if we leave in an old class method after it's added to the map

Reply:
This is a great check for now!